moveit_experimental/moveit_jog_arm/include/moveit_jog_arm/jog_calcs.h

```diff
@@ -142,7 +143,7 @@ class JogCalcs
 
   robot_state::RobotStatePtr kinematic_state_;
 
-  sensor_msgs::JointState joint_state_, original_joint_state_;
+  sensor_msgs::JointState incoming_joint_state_, internal_joint_state_, original_joint_state_;
```

Comment:
Does the `internal_joint_state_` correspond to a "current" joint state?

Reply:
What I mean is that it's used in calculations, it shouldn't be relied on to be accurate. Would there be a better way to say that?

I'm gonna try to get this list down to 2. It seems like incoming_joint_state_ and original_joint_state_ should be the same thing.

Reply:
It turns out that `original_joint_state_` is a little bit different from `incoming_joint_state_` so I left a comment to explain it. `incoming_joint_state_` has every joint, but passive joints might get dropped from `original_joint_state_`